mapper/mapper_test.go

```diff
@@ -19,4 +19,20 @@ func TestUnitMapper(t *testing.T) {
 		So(geoStartModel.Page.Breadcrumb[0].Title, ShouldEqual, "Home")
 		So(geoStartModel.Page.Breadcrumb[0].URI, ShouldEqual, "/")
 	})
+
+	Convey("test CreateAreaPage mapper correctly initialises an AreaModel", t, func() {
+		areaModel := CreateAreaPage(mdl)
+
+		So(areaModel.BetaBannerEnabled, ShouldBeTrue)
+		So(areaModel.Metadata.Title, ShouldEqual, areaModel.Name + " Summary")
+		So(len(areaModel.Page.Breadcrumb), ShouldEqual, len(areaModel.Ancestors) + 2)
+		So(areaModel.Page.Breadcrumb[0].Title, ShouldEqual, "Home")
+		So(areaModel.Page.Breadcrumb[0].URI, ShouldEqual, "/")
+		So(areaModel.Page.Breadcrumb[1].Title, ShouldEqual, "Areas")
+		So(areaModel.Page.Breadcrumb[1].URI, ShouldEqual, "/areas")
+		if len(areaModel.Ancestors) > 0 {
```

Comment:
Unusual to have an `if` in a unit test. This conditional isn't being hit by the setup of your test. You should expand your `areaModel` to have the properties so that you guarantee that the values are mapped correctly.

Reply:
I was trying to setup  the tests for when the model was expanded to load the data from the API, but it does make it confusing better to leave it out until that work is done.